src/main/java/com/avail/interpreter/levelTwo/operation/L2_MAKE_IMMUTABLE.java

```diff
@@ -149,16 +149,8 @@ public void instructionWasAdded (
 		final L2WriteBoxedOperand write = instruction.operand(1);
 
 		read.instructionWasAdded(manifest);
-
-		// Only deal with the boxed form.  The unboxed values are dealt with by
-		// subsequent move instructions.
-		final L2Synonym synonym =
-			manifest.semanticValueToSynonym(read.semanticValue());
-
-		// Make inaccessible all places holding the mutable boxed value.
-		manifest.forget(synonym);
-
-		write.instructionWasAdded(manifest);
+		write.instructionWasAddedForMakeImmutable(
```

Comment:
Pesky `makeImmutable`, needing its own reaction method.

Reply:
Yes, that thread of "specialness" continues to wind through L2WriteOperand and L2ValueManifest.